source/checkpoint_restart.f90

```diff
@@ -109,6 +110,75 @@ subroutine cr_expand_arrays(npart_new)
 
 end subroutine cr_expand_arrays
 
+! ================================================================================================ !
+!  CR KILL SWITCH
+!  V.K. Berglyd Olsen, BE-ABP-HSS
+!  Last modified: 2018-11-16
+!  This routine will kill SixTrack if the current turn number matches a number in crkillturns
+! ================================================================================================ !
+subroutine cr_killSwitch(iTurn)
```

Comment:
If I correctly understand, the call to this routine is at each turn... maybe we can already target the call to the correct turn...

Reply:
I fixed it by checking if we are on a kill turn before opening files etc.. Anyway this function is not called if the CRKILLSWITCH is `.true.`